src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -26,8 +26,10 @@ if (PHP_VERSION_ID >= 70200) {
     $PHPUNIT_VERSION = '4.8';
 }
 
+COMPOSER_FILE_NAME = getenv('COMPOSER_FILE_NAME') ? : 'composer.json';
```

Comment:
COMPOSER_FILE_NAME` is non-standard. According to https://getcomposer.org/doc/03-cli.md#environment-variables, it should be `COMPOSER` instead.

Reply:
This should work:
```
if ('composer.json' !== $COMPOSER_FILE_NAME = getenv('COMPOSER') ?: 'composer.json') {
    putenv('COMPOSER=composer.json');
}
```